softdev.bib

```diff
@@ -2300,3 +2300,40 @@ @article{medeiros19syntax
     year = 2019,
     url = {https://arxiv.org/abs/1905.02145}
 }
+
+@book{vinju05typedriven,
```

Comment:
Is this a PhD thesis?

Reply:
To be honest, I'm not really sure. Seems it's just a self-published article:
https://homepages.cwi.nl/~jurgenv/papers/RISE-2004.pdf

Reply:
It's almost certainly either a tech report (which might be a version of a paper that wasn't otherwise published; or an exact copy of a published paper) or a "real" paper. Might be worth searching around to see if/where it was formally published.

Reply:
I had a look but Google Scholar isn't helping here. The only thing I could find was a Conference paper citing it as `Vinju, J.J.: A type driven approach to concrete meta programming. Technical Report SEN-E0507, Centrum voor Wiskunde`

Reply:
Though, not sure where they got that tech report number from.

Reply:
You've picked up the trail :) Keep looking, I bet you can find it! At the very least you should be able to double check that tech report number with CWI's website.

Reply:
Okay, that citation is from one of Jurgen's paper's so I can assume this citation is the correct one. Tech report it is. :-)

Reply:
Fixed in 75d0e43

Reply:
Just to double check, can you find this on CWI's website?

[I've seen people cite their papers incorrectly before...]

Reply:
The paper, yes: https://homepages.cwi.nl/~jurgenv/papers/RISE-2004.pdf
The tech report number I managed to double check from this pdf here:
http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.106.5314&rep=rep1&type=pdf
Unfortunately, I couldn't find any reference directly from CWI's website.

Reply:
It's https://ir.cwi.nl/pub/10970, right?

Reply:
You are too good. I never thought to try to use their internal search, or did google give you this?

Reply:
Google, 3rd link ;)

Reply:
Which search terms, it still doesn't show up for me. :-(

Reply:
"cwi SEN-E0507" was the first thing I tried.

Reply:
Doesn't work for me, unfortunately.

Reply:
Oh well. We've at least found the source of truth, which is the main thing. But I would have tried CWI's internal search as my next step if Google hadn't pointed me in the right direction.

Reply:
This is a skill where I have still stuff to learn, I guess.